src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  width: 198px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  font-weight: 400;
+  border-radius: 5px;
+  padding-bottom: 16px;
+  margin-right: 50px;
+
+  &__section {
+    width: 166px;
+    padding: 0 14px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    padding: 31px 0 40px;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-dark-blue;
+    margin: 0;
+  }
+
+  &__product-number {
+    font-size: 10px;
+    line-height: 14px;
+    padding: 4px 0 0;
+    color: $color-grey;
+  }

```

Comment:
Use nesting for classes in scss. Fix it everywhere
```suggestion
  &__product {
  	&-name {
    font-size: 12px;
    font-weight: 500;
    line-height: 18px;
    color: $color-dark-blue;
    margin: 0;
  }
  
  &-number {
    font-size: 10px;
    line-height: 14px;
    padding: 4px 0 0;
    color: $color-grey;
   }
 }
```

![image](https://user-images.githubusercontent.com/70009205/225949296-7ec404b3-7d8f-44d6-b465-a902924a5dfa.png)